jina/serve/executors/__init__.py

```diff
@@ -124,7 +122,12 @@ def _add_runtime_args(self, _runtime_args: Optional[Dict]):
             self.runtime_args = SimpleNamespace()
 
     def _init_monitoring(self):
-        if hasattr(self.runtime_args, 'metrics_registry'):
+        if (
+            hasattr(self.runtime_args, 'metrics_registry')
```

Comment:
can't we make sure `metrics_registry` is always in the `runtime_args`?

Reply:
Yes we should you are right

Reply:
then let's do it here